docs/src/literate-tutorials/incompressible_elasticity.jl

```diff
@@ -258,7 +258,7 @@ function solve(ν, interpolation_u, interpolation_p)
     cellvalues_u, cellvalues_p, facevalues_u = create_values(interpolation_u, interpolation_p)
 
     ## Assembly and solve
-    K = create_sparsity_pattern(dh)
+    K = create_matrix(dh)
```

Comment:
Maybe we can showcase the use of the extension here?

Reply:
Following the discussion https://github.com/Ferrite-FEM/Ferrite.jl/pull/872#discussion_r1502656633 - I think it would be better to add such a showcase in a how-to or another tutorial, to avoid complicating the first tutorial on multi-field problems.

Reply:
I really have memory like a goldfish haha

Reply:
The block array extension? I don't think it make sense to use that unless you have a solver that takes advantage over easy access to individual blocks.